docs/reference/module-api/module-overview.md

```diff
@@ -720,6 +741,25 @@ module.exports = {
 };
 ```
 
+If including a route option, use an object. See the [route options](#route-options) section for more.
+
+Each route function takes the Express arguments `req` (the [request object](https://expressjs.com/en/api.html#req)) and `res` (the [response object](https://expressjs.com/en/api.html#res)). The functions must generate a response via `res` to avoid leaking resources, typically using the `res.redirect` or `res.send` methods.
+
+See [Naming routes](#naming-routes) for more on function names and their route URLs.
+
+::: tip
+We recommend using `apiRoutes` or `restApiRoutes` whenever possible before using `routes` as they handle the potential pitfalls of Express routes. There are situations where writing Express routes may be necessary, such as when you need to use `res.redirect` or pipe a stream.
+:::
+
+### Route options
+
+A route in the `apiRoutes`, `restApiRoutes`, `renderRoutes`, and `routes` sections (as well as their `extend` variations) may include options to govern particular behavior. If so, it should be configured as an object rather than a function. That object should include a `route` property in addition to the options:
+
+| Property | What is it? |
+| -------- | ----------- |
+| `route` | A route function that accepts a request as its argument |
+| `before` | Identify a module's middleware or routes. Used if the route function should be registered before particular middleware or other routes. Format as `middleware:nameOfModule` for middleware or `nameOfModule` for routes. |
```

Comment:
I would say "Identify another module's middleware or routes," I think it adds clarity about what this is intended to do.